query/language/undefine.feature

```diff
@@ -353,140 +516,195 @@ Feature: TypeQL Undefine Query
       | label:contract-employment |
     When typeql schema query
       """
-      undefine employment owns start-date;
+      undefine owns start-date from employment;
       """
     Then transaction commits
 
-    When session opens transaction of type: read
-    When get answers of typeql read query
+    When connection open read transaction for database: typedb
+    Then typeql read query; fails with a message containing: "empty-set for some variable"
       """
       match $x owns start-date;
       """
-    Then answer size is: 0
 
-
-  Scenario: removing key ownerships from a super relation type also removes them from its subtypes
-    Given typeql schema query
+# TODO: match with annotations (do we really need this test in this file? Only for undefine + match purposes...)
```

Comment:
TODO number 1, but I'm not sure if we want to keep this test here at all. I might completely remove it in favor of concept BDDs.